jest.preset.js

```diff
@@ -0,0 +1,5 @@
+const nxPreset = require('@nrwl/jest/preset').default
+
+console.log(nxPreset)
```

Comment:
Consciously a console.log() here?

Reply:
Yeah saw that in the meeting haha, gonna remove it with this
```suggestion

```